tests/Aspire.Cli.Tests/Hosting/CliBackchannelTests.cs

```diff
@@ -50,6 +51,7 @@ public async Task AppHostRespondsToPingWithMatchingTimestamp()
 
     [Fact]
     [ActiveIssue("https://github.com/dotnet/aspire/issues/8113")]
+    [QuarantinedTest("https://github.com/dotnet/aspire/issues/8113")]
```

Comment:
What happens when a test is both ActiveIssue and QuarantinedTest? Is there an intention of separating a test that is flaky vs a test that is just failing consistently? The original discussion mentioned that there was value in separating plain failing tests vs flaky ones, and having the flaky use QuarantinedTest and failing tests using ActiveIssue.

Reply:
Not saying this is how it has to work, I'm interested in what you think we should do here.

Reply:
That's a good question.

I thought these two can conicide, but after some tests (and some lost hair) I came to a conclusion that, essentially, these two attributes clash. `ActiveIssue` appears to act as a conditional attribute by skipping tests for a specified platform. And this prevents tests to be executed in the outerloop.